docs/docs/examples/guides/05-jtc-guide.md

```diff
@@ -0,0 +1,602 @@
+
+<div class="text--center">
+  <img src={jtcGuideParameters} alt="Basic JTC parameters" />
+</div>
+
+Let us cover some of these parameters:
+
+- `Command interfaces`: that refers to the type of command that JTC will use to control your robot; one of position,
+velocity, acceleration, or effort.
+- `State interfaces`: the type of state interfaces JTC will claim. Some interfaces may be necessary depending on your
+choice of command interfaces. For example, if you want to control your robot in acceleration, you need to have a
+velocity state interface. 
+- `Open loop control`: refers to whether the controller will be correcting its desired commands according to the state
+it is receiving or not.
+- `Allow nonzero velocity at trajectory end`: typically, you want the robot to halt when it reaches the end of your
+trajectory. However, there are cases, e.g., when you are controlling in velocities, where you want to allow non-zero
+values even if the trajectory waypoints have been successfully traversed.
+- `Proportional/Integral/Derivative gain(s)`: these gains are to be set and tuned when you are, for example, commanding
+in velocity but your trajectory has positions. Then, a PID control will be applied to generate desired velocities
+through the trajectory.
+
+You will also notice tolerance values for trajectory execution times and position offsets. These should be set on a
+per-application basis, as failing to satisfy the corresponding constraints would lead to the trajectory failing
+mid-execution.
+
+You may find advanced parameters in the `Docs` tab under the `Joint Trajectory Controller` page. These parameters can be
+used to further tune JTC's performance and functionality, but may also require more advanced knowledge in order to tune
+them. Make sure to carefully read the descriptions next to each parameter before making changes.
+
+### Setting a trajectory for execution
+
+There are 2 ways of setting a trajectory in JTC:
+
+1. by using a `JointTrajectory` signal
+2. by calling the `set_trajectory` service
+
+In both cases, receiving a new joint trajectory will first trigger cancellation of an active trajectory, if there is
+one. That is, **there is no trajectory buffering or appending taking place**. As with many things in the AICA Universe,
+behaviors are event-driven. If you wish to send multiple trajectories back-to-back, you will have to rely on the
+execution status of the active trajectory handled by JTC. There is a practical example of how do this in following
+sections (see [Putting an application together](#putting-an-application-together)). 
+
+### Trajectory execution status
+
+The controller exposes 4 predicates to reflect the the execution status of a trajectory, namely:
+
+- `Has active trajectory`: A trajectory has been set and is being executed
+- `Has trajectory succeeded`: A trajectory was executed successfully (i.e., reached the final waypoint respecting all 
+tolerances in the way)
```

Comment:
This sentence is a bit weird, maybe:
```suggestion
tolerances on the way)
```

Or just "reached the final waypoint within all tolerances"